_layouts/eip.html

```diff
@@ -41,4 +41,31 @@ <h1 class="page-heading">
 
   {{ content }}
 
+  <h2>Citation</h2>
+  <p>Please cite this document as:</p>
+  {% comment %}
+  IEEE specification for reference formatting:
+  https://ieee-dataport.org/sites/default/files/analysis/27/IEEE%20Citation%20Guidelines.pdf
+  {% endcomment %}
+  <p>{% include authorlist.html authors=page.author %}, "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status != "Final" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
 </div>
+
+{% comment %}
+Article schema specification:
+https://schema.org/Article
+{% endcomment %}
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "author": "{% include authorlist.html authors=page.author %}",
+    "name": "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status != "Final" %} [DRAFT]{% endif %}",
+    "dateCreated": "{{ page.created | date: "%Y-%m-%d" }}",
+{% if page["discussions-to"] != undefined %}
+    "discussionUrl": "{{ page["discussions-to"] | uri_escape }}",
+{% endif %}    
+    "inLanguage": "en-US",
```

Comment:
This should be also mandated in EIP-1.

Reply:
That language be English?

I agree. But I'm hoping to get this PR merged this year so I avoided touching EIP-1. I hope this PR is mergeable as-is and then updates to EIP-1 can be considered separately and next.